src/data/supervisor/update.ts

```diff
@@ -0,0 +1,21 @@
+import type { HomeAssistant } from "../../types";
+
+export interface SupervisorUpdateConfig {
+  add_on_backup_before_update: boolean;
+  add_on_backup_retain_copies?: number;
+  core_backup_before_update: boolean;
+}
+
+export const getConfig = async (hass: HomeAssistant) =>
```

Comment:
Make this name a little more specific, so in the imported file I know what config is being get.